resources/leiningen/new/luminus/keeframe/src/cljs/routing.cljs

```diff
@@ -1,65 +1,20 @@
 (ns <<project-ns>>.routing
   (:require
-    [kee-frame.api :as api]
     [re-frame.core :as rf]
-    [reitit.core :as reitit]
-    [clojure.string :as string]
-    [goog.events :as events]
-    [goog.history.EventType :as HistoryEventType])
+    [reitit.core :as reitit])
   (:import goog.History))
 
 (def routes
   [["/" :home]
    ["/about" :about]])
 
-(defonce router (reitit/router routes))
-
-(defrecord ReititRouter [routes]
```

Comment:
Implementation of kee-frame is now pretty much stolen from the code deleted in this PR. So I guess it's safe to use that instead? I could have another look to see if they are in fact identical.

Reply:
Yeah it looks like it should work the same. You can quickly test if you spin up a new app with `+kee-frame` to see if the default routes are working.